packages/camera/camera_avfoundation/ios/Classes/FLTCamMediaSettings.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTCamMediaSettings.h"
+
+@implementation FLTCamMediaSettings
+
+#define AssertPositiveNumberOrNil(param)                                                        \
+  if (param != nil) {                                                                           \
+    NSAssert([param isKindOfClass:NSNumber.class], @"%@ is not a number: %@", @ #param, param); \
```

Comment:
very minor nit: `+class` is a method not a class property so shouldn't use dot notation.
```suggestion
    NSAssert([param isKindOfClass:[NSNumber class]], @"%@ is not a number: %@", @ #param, param); \
```